### ISSUE TITLE
Small Log10P values

### ISSUE THREAD

**Issue by Antonio-Nappi** [rgcgithub/regenie#385]
TL;DR
Extremely low pvalues in burden test compared with the ones in Genebass analysis, why?

Hello to everyone, I am using Regenie(version  v3.2.3) to perform burden test. Right now I am using BMI as pheno just to validate my pipeline. I am working with UKBB data. To speed up the validation of the pipeline I am using the _calls_ files, rather than the imputed one. First of all I filtered my data as reported in the documentation
```
plink --bed ~/ukb22418_c22_b0_v2.bed --fam ~/ukb22418_c22_b0_v2_s488170.fam --bim ~/ukb_snp_chr22_v2.bim --keep ~/Regenie_to.keep --out ~/calls-file-filtered/calls_chrom22_filtered --maf 0.01 --make-bed --mac 10 --geno 0.1 --hwe 1e-15
```
where the file `Regenie_to.keep` contains the `FID` and `IID` of european people in ukbb not related (so `FID` and `IID` are equal).
I have done this operation for all the chromosomes.
Once finished I have run this command to merge all the files processed in the previous stage:
```
plink --make-bed --merge-list ~/calls-file-filtered/list_to_merge.txt --out ~/calls-file-filtered/calls_chrom_whole_filtered
```
the file `list_to_merge.txt` contains the path to each processed chrom:
```
~/calls-file-filtered/calls_chrom1_filtered
~/calls-file-filtered/calls_chrom2_filtered
~/calls-file-filtered/calls_chrom3_filtered
~/calls-file-filtered/calls_chrom4_filtered
~/calls-file-filtered/calls_chrom5_filtered
```
Last preprocess step was to convert the bed file obtained into a bgen-1.2, because Regenie is optimized for this format.
```
plink2 --bfile ~/calls-file-filtered/ukb_calls_merged_1_22_v3_filtered_hg38 --export bgen-1.2 "bits=8" --out ~/bgen_ukb_calls_merged_1_22_v3_filtered_hg38
```
Finally I can run the step 1 of Regenie with the following command:
```
regenie \
--step 1 \
--bgen ~/bgen_ukb_calls_merged_1_22_v3_filtered_hg38.bgen \
--covarFile~/covs_qced_eur_tab_sep.tsv  \
--phenoFile ~/regenie_pheno.tsv \
--bsize 200 \
--out ~/Regenie_output/test_plof_only_8.02_whole_genome_imp_bgen \
--iid-only \
--force-step1'''
```
I had to specify `--force-step1` since the file contains >1M variants. Both the covar file and the pheno file are tab separeted values files.
Then I prepared the files for the step 2:
the set-list file:
```
ZNF595	4	85731	4:85731:C:T,4:85733:A:G,...
ZNF718	4	124673	4:124673:G:C
ZNF732	4	271103	4:271103:A:T,4:271106:T:C,...
```
the annotation file:
```
4:85731:C:T	ZNF595	missense
4:85733:A:G	ZNF595	missense
4:85734:T:TA	ZNF595	missense
4:85737:G:A	ZNF595	missense
4:85737:G:C	ZNF595	missense
4:85741:T:C	ZNF595	synonymous
4:85742:C:T	ZNF595	missense
4:85753:A:T	ZNF595	missense
```
and the mask definition file (I am focusing my analysis on the pLoF variants only at the moment):
```
Mask1 pLoF
```
The command to run the step 2 is the following:
```
regenie \
  --step 2 \
  --pgen ~/pgen/ukb23159_c4_b0_v1 \
  --covarFil~/covs_qced_eur_tab_sep.tsv \
  --phenoFile ~/regenie_pheno.tsv \
  --pred ~/Regenie_output/test_plof_only_8.02_whole_genome_calls_bgen_pred.list \
  --anno-file ~/pgen-filtered-maf0.01-eur/ukb23159_c4_b0_v1_antonio.pann \
  --set-list ~/ukb23159_c4_b0_v1_antonio_set_list.pvar \
  --mask-def ~/pgen-filtered-maf0.01-eur/mask_definition.txt \
  --write-mask \
  --bsize 200 \
  --out~/Regenie_output/test_plof_only_8.02_step_2_whole_genome_calls_bgen \
  --aaf-bins 0.01,0.005
```
The pgen file in input in the step 2 is the the result of the WES data.

Obtained my results I decided to compare them with the one present in Genebass([here](https://app.genebass.org/gene/undefined/phenotype/continuous-21001-both_sexes--irnt?resultIndex=gene-manhattan&resultLayout=full)) and what I've noticed is that my log10Pvalues are extremely low both in absolute value (the maximum one is ~3.5) and compared with the GeneBass ones (for gene KIAA1109 GeneBass has a burden pvalue ~10, while for the same gene, and singleton as mask, my pvalue is ~0.84 while with other thresholds this value is even lower). What am I doing wrong?

**Comment by Antonio-Nappi** [comment]
Found the error, there was an error mapping the variants in the wes data used for step 2
